src/tests/ftest/SConscript

```diff
@@ -0,0 +1,41 @@
+"""Build ftest"""
+import os
+
+def scons():
+    """Execute build"""
+    Import('env', 'prereqs')
+
+    ftest_install_dir = '$PREFIX/lib/daos/TESTING/ftest'
+
+    env.Install(ftest_install_dir, Glob('*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'container'),
+                 Glob('container/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'control'),
+                 Glob('control/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'daos_test'),
+                 Glob('daos_test/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'data'),
+                 Glob('data/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'io'),
+                 Glob('io/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'network'),
+                 Glob('network/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'object'),
+                 Glob('object/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'pool'),
+                 Glob('pool/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'rebuild'),
+                 Glob('rebuild/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'security'),
+                 Glob('security/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'server'),
+                 Glob('server/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'soak'),
+                 Glob('soak/*.*'))
+    env.Install(os.path.join(ftest_install_dir, 'unittest'),
+                 Glob('unittest/*.*'))
+    # util
+    SConscript('util/SConscript', exports=['env'])
```

Comment:
I don't believe you need the exports argument here.

Reply:
right, removed it.